activitysim/examples/example_mtc_extended/configs/vehicle_type_choice_op2_fuel_type_probs.csv

```diff
@@ -0,0 +1,101 @@
+vehicle_type,Gas,Diesel,Hybrid,PEV,BEV
+Car_1,.9205,0.0020,0.0468,0.0153,0.0155
+Car_2,.9095,0.0120,0.0546,0.0082,0.0157
+Car_3,.9038,0.0163,0.0595,0.0115,0.0088
+Car_4,.8951,0.0167,0.0684,0.0107,0.0090
+Car_5,.9079,0.0155,0.0646,0.0065,0.0054
+Car_6,.9365,0.0137,0.0436,0.0019,0.0043
+Car_7,.9131,0.0117,0.0744,0.0006,0.0002
+Car_8,.9541,0.0096,0.0355,0.0007,0.0000
+Car_9,.9548,0.0037,0.0409,0.0004,0.0001
+Car_10,0.9530,0.0015,0.0451,0.0003,0.0001
+Car_11,0.9676,0.0096,0.0225,0.0003,0.0000
```

Comment:
It seems as though some amount of "smoothing" would make sense here between the years.  Many of the numbers for the lower-probability options go back and forth between 0 and >0 several times.

Reply:
Yes, these probabilities are "lumpy".  This is due to the fact that they came directly from the National Household Travel Survey data.  A comment about the "lumpy-ness" was added to the models.rst vehicle type choice documentation encouraging the user to change the probabilities to their region and smooth them as they see fit.  Additional links in the documentation point to the presented results and sensitivity studies.

So, I suggest we leave the probabilities as is in the context of this pull request for the purpose of simplicity and transparency.